src/app/features/video-list/item/item.component.ts

```diff
@@ -41,6 +43,10 @@ export class ItemComponent implements OnInit {
     this.viewCount = this.video.viewCount;
     this.favorite = this.video.favorite;
     this.favoriteSwitch = this.video.favorite ? MaterialIcons.favorite : MaterialIcons.favorite_outline;
+
+    this.snackBar.open(Messages.dodales_film_do_ulubionych, Messages.zamknij, {
+      duration: 5000,
+    });
```

Comment:
Messages.dodales_film_do_ulubionych najlepiej klucz w angielskim napisać i 5000 to taki magic number